src/NLog.Targets.MauiLog/MauiLogTarget.MaciOS.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Diagnostics;
+using Layout = NLog.Layouts.Layout;
+
+namespace NLog.Targets;
+
+/// <summary>
+/// Output target for System.Diagnostics.Debugger.Log with fallback to Console when no debugger
+/// </summary>
+
+[Target("MauiLog")]
+[NLog.Targets.MauiLogLinker.Preserve]
+
+public class MauiLogTarget : TargetWithLayoutHeaderAndFooter
```

Comment:
Possible that the new namespace-behavior can be reverted, so it will be git-rename, instead git-delete and git-rename.